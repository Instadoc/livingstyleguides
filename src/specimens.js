const html = require("./specimens/html")

module.exports = [html()]
