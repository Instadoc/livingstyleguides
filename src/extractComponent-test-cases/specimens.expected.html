<p>This is the first specimen:</p>
<iframe src="Component Name/specimen-1.html"></iframe>
<pre><code class="language-specimen-1.html">&#x3C;b>Specimen 1&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-1.css">b { color: red }
</code></pre>
<p>These are not specimens:</p>
<pre><code class="language-html">&#x3C;b>Not a specimen&#x3C;/b>
</code></pre>
<pre><code>Also not a specimen
</code></pre>
<p>This is the second specimen:</p>
<iframe src="Component Name/specimen-2.html"></iframe>
<pre><code class="language-specimen-2.html">&#x3C;b>Specimen 2&#x3C;/b>
</code></pre>
<pre><code class="language-specimen-2.css">b { color: green }
</code></pre>